integration_tests/test_gravity.py

```diff
@@ -39,11 +38,7 @@ def cronos_crc20_abi():
 
 def gorc_config(keystore, gravity_contract, eth_rpc, cosmos_grpc, metrics_listen):
     return {
-        "keystore": InlineTable(
-            {
-                "File": str(keystore),
-            }
-        ),
+        "keystore": str(keystore),
```

Comment:
will need to change back after aws secret manage feature is merged.

Reply:
which repo will integrate the `aws secret manager` feature? Is the `gravity-bridge`?